src/assets/Directory.js

```diff
@@ -0,0 +1,149 @@
+                        src={user.profile_picture}
+                        size="large"
+                        style={{ marginRight: '10px' }}
+                      />
+                      <Link to={`/user/${user.id}`}>{user.display_name}</Link>
+                    </>
+                  }
+                  extra={
+                    props.user.id !== user.id && (
+                      <>
+                        <Link
+                          to={`/message/send/${props.user.id}/receive/${user.id}`}
+                        >
+                          <Button
+                            type="primary"
+                            style={{ width: '100px', marginRight: '10px' }}
+                          >
+                            Message
+                          </Button>
+                        </Link>
+                        {isFollowing(user.id) ? (
+                          <Button
+                            type="primary"
+                            style={{ width: '100px' }}
+                            onClick={() => {
+                              props
+                                .unfollow(props.user.id, user.id)
+                                .then(() => props.fetchUser());
+                            }}
+                          >
+                            Unfollow
+                          </Button>
+                        ) : (
+                          <Button
+                            type="primary"
+                            style={{ width: '100px' }}
+                            onClick={() => {
+                              props
+                                .follow(props.user.id, user.id)
+                                .then(() => props.fetchUser());
+                            }}
+                          >
+                            Follow
+                          </Button>
+                        )}
+                      </>
+                    )
+                  }
+                >
+                  {
+                    <div
+                      style={{
+                        display: 'flex',
+                        justifyContent: 'space-between',
+                      }}
+                    >
+                      <span>
+                        <b style={{ marginRight: '5px' }}>Track:</b>
+                        {`${user.track.toUpperCase()}`}
+                      </span>
+                      <span>
+                        <b style={{ marginRight: '5px' }}>Following:</b>
+                        {`${user.following}`}
+                      </span>
+                      <span>
+                        {user.github_username ? (
+                          <a
+                            href={`https://github.com/${user.github_username}`}
+                          >
+                            <b>Github</b>
+                          </a>
+                        ) : (
+                          <b>Github</b>
```

Comment:
Maybe we can change this to just be blank if there is no GitHub username provided by the user

Reply:
I'll probably make this change before you when I go into the file to move over the styles

Reply:
I'll change it back if it doesn't look right

Reply:
You can modify this as you think about styling the component. Currently this is how it looks, black if no link and blue if it's linked. 
![image](https://user-images.githubusercontent.com/5246983/115959825-d0731a80-a516-11eb-9677-4519e92357be.png)

Reply:
Ah I see, I'll think about the styling of this directory this week.